### DISCUSSION TITLE
SSC8339D datasheet

### DISCUSSION THREAD

**Discussion by bkleiner** [linux-chenxing/linux-chenxing.org#68] [General]
Hi,
it appears the SSC8339D is used in the HDZero VRX https://repair.wiki/w/HDZero_VRX
does anybody have any datasheets for this SoC?
there is some mention in the GH pages.

**Comment by fifteenhex** [comment]
There's no datasheet but it's a "mercury5" chip.

http://linux-chenxing.org/mercury5/#pinout
